server/node/src/main/java/org/kaaproject/kaa/server/control/service/sdk/JavaSdkGenerator.java

```diff
@@ -636,7 +636,7 @@ private void packageSources(List<JavaDynamicBean> javaSources, Map<String, ZipEn
         JavaDynamicCompiler dynamicCompiler = new JavaDynamicCompiler();
         dynamicCompiler.init();
         for (JavaDynamicBean bean : javaSources) {
-            LOG.debug("Compiling bean {} with source: {}", bean.getName(), bean.getCharContent(true));
+            LOG.trace("Compiling bean [{}] with source:\n{}", bean.getName(), bean.getCharContent(true));
```

Comment:
Maybe 
LOG.debug("Compiling bean: {}", bean.getName());
LOG.trace("Bean source:\n {}", bean.getCharContent(true));
is better?